pkg/service/manipulator_test.go

```diff
@@ -84,7 +84,18 @@ func TestManipulator_Process(t *testing.T) {
 	})
 	assert.Nil(t, err)
 	assert.Equal(t, input, data)
+	assert.True(t, mp.AssertNotCalled(t, "FixOrientation", decoded, 0))
```

Comment:
I don't think we need this, because it will fail by nature of mock if it gets called before we put this line of code: `mp.On("FixOrientation", decoded, mock.Anything).Return(decoded)`

Reply:
ok, removed